src/main/resources/archetype-resources/silo/src/main/resources/db/common/V1__schema.sql

```diff
@@ -0,0 +1,4 @@
+-- THIS IS AN EXAMPLE - THINK BEFORE YOU USE IT
```

Comment:
Because we dont know (if its possible) how to create empty folder with the archetype this is only a dummy file to have the folder common created.